security/pkg/pki/ca/ca_test.go

```diff
@@ -392,3 +429,14 @@ func createSecret(namespace, signingCert, signingKey, rootCert string) *v1.Secre
 		Type: istioCASecretType,
 	}
 }
+
+func comparePem(expectedBytes []byte, file string) error {
+	fileBytes, err := ioutil.ReadFile(file)
+	if err != nil {
```

Comment:
This fails only when the test setup is bad (i.e. test file is missing), which would fail your construction earlier. I rather ignore this error for simplicity. Function can just return boolean.

Reply:
OK. That makes sense. I can modify it.